README.md

```diff
@@ -539,6 +541,83 @@ module3 *
 Inclusion and exclusion files can be used together, in that case inclusion
 takes precedence over exclusion.
 
+If you need to set a custom precision for a set of call sites, you can use the functions's instrumentation proposed by verificarlo. Firstly, you need to activate instrumentation by giving the `--inst-func` flag to verificarlo:
```

Comment:
Create a specific section in the README.

Reply:
I've made a specific section for the custom precision too